pytorch_lightning/callbacks/__init__.py

```diff
@@ -35,4 +36,6 @@
     'ProgressBar',
     'ProgressBarBase',
     'ModelPruning',
```

Comment:
Could you make sure these are in alphabetical order (seems only `ModelPruning` is out of place)

Reply:
probably in the next iteration and we shall do it in all `__all__`